charter-proposals/charter.md

```diff
@@ -1,19 +1,18 @@
 
-By July 2020, some 200 models in SDF format have been contributed by participating ecosystems; new models are being submitted continually. Version 1.0 of the SDF specification was published on the OneDM github repository and as an Internet-Draft. OneDM is now focusing on consolidating the body of submitted models and developing processes for arriving at harmonized models that span different industry ecosystems in a common way; as expressed at the IETF 108 ASDF BoF, they look to the IETF for developing SDF 1.0 further into a high-quality specification.
-
-The IETF also defines the YANG data modelling language (RFC 7950). YANG has a strong focus on modelling the management interface for network devices, using a fairly small set of network management protocols, whereas SDF is designed to span (and transitively unify) many existing interaction models already present. While data associated with a YANG schema can be serialized to a variety of wire formats (e.g., XML, JSON, CBOR, and others), the serialization is driven directly by the structure of the data models defined in YANG; there is no separate "protocol binding" step as in the use of SDF.
-
-Conversely, SDF does not deal directly with serialization at all, modelling only the structure and semantics of the data being interchanged, hence leaving data serialization (and RPC semantics) to other standards, most likely defined by existing IoT SDOs.
+SDF does not deal directly with serialization at all, modelling only the structure and semantics of the data being interchanged, hence leaving data serialization (and RPC semantics) to other standards, most likely defined by existing IoT SDOs.
 
 The ASDF Working Group
 ----------------------
 
-The objective of the ASDF working group is to develop SDF into a standards-track specification for thing interaction and data modelling. In the process of developing this specification, further functional requirements that emerge in the usage of SDF for model harmonization will be addressed.
+The ASDF has developped SDF into a standards-track specification for thing interaction and data modelling.  In the process of developing this specification, further functional requirements have emerged that can be addressed as extensions to the base SDF specification.
+
+The ASDF WG will work with experts from OneDM and its contributing organizations to extend SDF to cover aspects such as digital twin, mapping to other IoT SDOs, and gateway interactions translations between IP and other transports.
```

Comment:
```suggestion
The ASDF WG will work with experts from OneDM and its contributing organizations to extend SDF to cover aspects such as digital twin, mapping to other IoT SDOs, and gateway interactions translating between IP and other transports.
```

Reply:
I think this is a good proposed draft for the charter, I think we can take it to the working group for discussion now